CleverPush/Source/CleverPushInstance.m

```diff
@@ -462,9 +463,20 @@ - (id)initWithLaunchOptions:(NSDictionary* _Nullable)launchOptions
         [userDefaults synchronize];
     }
 
-    NSDictionary* userInfo = [launchOptions objectForKey:UIApplicationLaunchOptionsRemoteNotificationKey];
-    if (userInfo) {
-        startFromNotification = YES;
+    if (launchOptions == nil) {
+        launchOptions = [CleverPushSelectorHelpers getStoredLaunchOptions];
+    }
+    
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(onDidFinishLaunchingNotification:)
```

Comment:
Do we still need that if we have the swizzling already?
Rest looks good to me!

Reply:
@gbhrdt,<br>PR had been updated.